portfolio/target/portfolio-1/calendar.js

```diff
@@ -0,0 +1,51 @@
+class CalendarEvent{
+  constructor(name, startTime, endTime) {
```

Comment:
Similar comment about using Date instead of timestamp variables.

Reply:
Updated, although in future PR these files under target/ will be ignored.